### PR TITLE
Redesign fixes 27 aug

### PR DESCRIPTION
**Pull request by ryaplots**
#### Summary

This PR adds a batch of fixes to the redesign

#### Changes

- Add back verbose stream switch to events
- Remove cookie state and use persisted state for pageSize
- Add visual indication of language
- Fix map height
- Fix routing policy style

#### Checklist

- [x] Scope: The referenced issue is addressed, there are no unrelated changes.
- [x] Compatibility: The changes are backwards compatible with existing API, storage, configuration and CLI, according to the compatibility commitments in `README.md` for the chosen target branch.
- [ ] Documentation: Relevant documentation is added or updated.
- [ ] Testing: The steps/process to test this feature are clearly explained including testing for regressions.
- [ ] Infrastructure: If infrastructural changes (e.g., new RPC, configuration) are needed, a separate issue is created in the infrastructural repositories.
- [ ] Changelog: Significant features, behavior changes, deprecations and fixes are added to `CHANGELOG.md`.
- [x] Commits: Commit messages follow guidelines in `CONTRIBUTING.md`, there are no fixup commits left.

### CODE REVIEW

File: pkg/webui/components/map/map.styl [hunk 1]
```diff
@@ -16,6 +16,7 @@
   border-normal()
   width: 100%
   height: 100%
+  z-index: 'calc(%s - 2)' % ($zi.dropdown)
```
Comment (kschiffer) [inline pkg/webui/components/map/map.styl]:
```suggestion
  z-index: $zi.dropdown - 2
```
?

File: pkg/webui/constants/colors.js [hunk 2]
```diff
@@ -13,9 +13,9 @@
 // limitations under the License.
 
 export default Object.freeze({
-  C_ACTIVE_BLUE: '#3D73FF',
-  C_ACTIVE_BLUE_HOVER: '#1E5DFF',
-  C_ACTIVE_BLUE_ACTIVE: '#0047FF',
+  C_ACTIVE_BLUE: '#1E5DFF',
+  C_ACTIVE_BLUE_HOVER: '#1547DB',
+  C_ACTIVE_BLUE_ACTIVE: '#0F34B7',
   C_ICON_FILL: '#686868',
```
Comment (kschiffer) [inline pkg/webui/constants/colors.js]: Please also update the constant names so they align with the token names.

File: pkg/webui/console/components/events/index.js [hunk 3]
```diff
@@ -30,6 +30,7 @@ import Icon, {
   IconClipboardCheck,
 } from '@ttn-lw/components/icon'
 import Button from '@ttn-lw/components/button'
+import Routes from '@ttn-lw/components/switch'
```
Comment (kschiffer) [inline pkg/webui/console/components/events/index.js]:
```suggestion
import Switch from '@ttn-lw/components/switch'
```
That confused me good 😂

File: pkg/webui/console/components/events/index.js [hunk 4]
```diff
@@ -165,6 +173,12 @@ const Events = React.memo(
             <div className={style.stickyContainer}>
               <div className={style.actions}>
                 <>
+                  {!disableFiltering && (
```
Comment (kschiffer) [inline pkg/webui/console/components/events/index.js:176]: Let's add it to both normal event view and split view events.

**Review by kschiffer — CHANGES_REQUESTED** [review]

**Comment by kschiffer** [comment]
Also the global page size doesn't seem to persist now.

File: pkg/webui/constants/colors.js (hunk 2 above)
Comment (ryaplots) [reply pkg/webui/constants/colors.js]: The rest of the colors are actually not used anywhere, so we can remove them?

File: pkg/webui/components/map/map.styl (hunk 1 above)
Comment (ryaplots) [reply pkg/webui/components/map/map.styl]: indeed, no need to make it harder than it has to be.

File: pkg/webui/constants/colors.js (hunk 2 above)
Comment (kschiffer) [reply pkg/webui/constants/colors.js]: Yes, let's remove them then.

**Review by kschiffer — APPROVED** [review]
CI seems like a flake?

**Comment by ryaplots** [comment]
> CI seems like a flake?

Yeah this happens sometimes, where the previous run is not closed, but there a second run attempt

**Pull request merged**